src/client-configuration.ts

```diff
@@ -19,6 +20,41 @@ export interface ClientConfiguration {
    */
   endpoint?: URL;
 
+  /**
+   * When true will keep executing a request even if the page
+   * that fired the request is no longer executing. Only relevant
+   * to underlying clients using the {@link https://fetch.spec.whatwg.org/ | Fetch standard}.
+   * By default set to false.
+   * @remarks
+   * Relevant to clients using the {@link FetchClient} provided,
+   * or any custom HTTP Clients you implement using the Fetch standard.
+   */
+  fetch_keepalive?: boolean;
```

Comment:
why did we move the lines?

Reply:
The fields used for default query options are all under the client-config-only fields